Week4/app/src/main/java/com/kulloveth/moviesapp/ui/movies/MoviesFragment.kt

```diff
@@ -57,52 +65,36 @@ class MoviesFragment : Fragment(), MovieAdapter.MovieItemCLickedListener {
         binding?.contentLayout?.toolbar?.title = getString(R.string.movies)
         adapter = MovieAdapter(this)
         recyclerView = binding?.contentLayout?.showMoviesRv
+        recyclerView?.layoutManager =
+            LinearLayoutManager(requireActivity())
+        recyclerView?.adapter = adapter
 
         (requireActivity() as AppCompatActivity?)?.setSupportActionBar(binding?.contentLayout?.toolbar)
+
         moviesDataManager = ViewModelProvider(requireActivity()).get(MoviesDataManager::class.java)
         bindMoviesRecyclerView()
-        val sharedPref = SignInRepository.sharedPrefs()
-        val userName =
-            sharedPref.getString(SignInFragment.USER_NAME_KEY, "")
-        val userImage = sharedPref.getString(SignInFragment.USER_IMAGE_KEY,"")
-        binding?.contentLayout?.toolbarImage?.let {
-            Glide.with(this).load(userImage).placeholder(R.drawable.ic_account_circle_white_24dp)
```

Comment:
Nice work using Glide!

Reply:
I had issues with Glide though so my final commits are using Picasso instead, even the Glide dependency is not there anymore